sci-geosciences/qmapshack/qmapshack-1.17.1_p20250426-r1.ebuild

```
@@ -34,18 +34,9 @@ DEPEND="${RDEPEND}"
 BDEPEND="dev-qt/qttools:6[linguist]"
 
 src_configure() {
-	local mycmakeargs=( -DUSE_QT6DBus=$(usex dbus) )
+	local mycmakeargs=(
+		-DUSE_QT6DBus=$(usex dbus)
+		-DHTML_INSTALL_DIR=/usr/share/doc/${PF}/html
```

```suggestion
		-DHTML_INSTALL_DIR="${EPREFIX}/usr/share/doc/${PF}/html"
```

... but isn't it the wrong path really? What is being installed does not look like html files to me at least.
Looking at kconfig:6, for one example, it is using Qt6 doc standard path:
```
--- /usr/share/qt6-doc/
>>> /usr/share/qt6-doc/KF6Config.qch
>>> /usr/share/qt6-doc/KF6Config.tags
```

... and while that's certainly \*a\* fix, their build system desperately needs GNUInstallDirs cmake module treatment.

---

Maybe `"${EPREFIX}/usr/share/doc/${PF}/qch"` could be an option, or plain into ...PF/

While this is Qt it does not use KDE cmake, so the variables mentioned are not available but could be used.

---

I know it is not using KDE cmake, I was just giving references. Basically though, what @ionenwks said.